### DISCUSSION TITLE
copilot automatic completion interfere with the indentation of the code

### DISCUSSION THREAD

**Discussion by yuriten** [community/community#65210] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Why does the copilot automatic completion interfere with the indentation of the code? 

![CleanShot 2023-08-29 at 00 11 19](https://github.com/community/community/assets/51118096/0bc9dca0-4627-46d1-b2d3-0c962b3bc044)

Every time you enter console.log, it will always lead to code deletion. Not only will the space be deleted, but it will also directly lead to grammatical errors. 

Once I disabled copilot extension, this error no longer appeared (and of course the autocompletion was gone too)

---
* vscode extension version: v1.105.350

**Comment by yuriten** [comment]
Auto-completion not only deletes two spaces, but also deletes "co", which directly leads to code errors

**Comment by arafajs** [comment]
Common problem, theses steps to try to fix it:

- Disable inline suggestions. 
- Use the sidebar instead of inline suggestions.
- Use a custom key binding for tab completion.
- Use a different autocompletion extension.

**Reply by yuriten** [reply]
But I do want to use the inline suggestions, I just don't want it to delete my code.

**Comment by AbdulazizElsafadi** [comment]
Is there a solution without disabling inline suggestions?
